### PR TITLE
Clean up Android setup docs #1250

### PR DESCRIPTION
**Pull request by pippinlee**
Clean up info for using ngrok, android developer bridge, and SDK for Android setup docs

📜:dizzy:

### CODE REVIEW

File: _docs/get_started/android.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Unfortunately, Webview in Android 4.2 does not ship with a debugger – we will
 npm install -g weinre
 ```
 
-In order to make debugging easier, you will also want to tunnel to the local port that `weinre` runs on (the alternative is making sure your local computer and phone are on the same network and looking up your IP address, which is time-consuming and requires constant re-configuration). I use `ngrok` to do this. You can download it from [the ngrok website](`ngrok`).
+In order to make debugging easier, you will also want to tunnel to the local port that `weinre` runs on (the alternative is making sure your local computer and phone are on the same network and looking up your IP address, which is time-consuming and requires constant re-configuration). I use `ngrok` to do this. You can download it from [the ngrok website](https://ngrok.com/).
```
Comment (thisandagain) [inline _docs/get_started/android.md]: Instead of "I use `ngrok` to do this" perhaps we could say, "You can use `ngrok`  to do this".

**Comment by thisandagain** [comment]
Looks great! One minor adjustment in the line notes.

File: _docs/get_started/android.md (hunk 1 above)
Comment (pippinlee) [inline _docs/get_started/android.md]: @thisandagain Wasn't sure if I should touch that phrasing of the copy -- but totally agree. Will update this PR.

**Comment by thisandagain** [comment]
\o/ R+ Thanks @pippinlee !

**Pull request merged**